packages/elements-dev-portal/package.json

```diff
@@ -47,8 +47,8 @@
   },
   "dependencies": {
     "@fortawesome/free-solid-svg-icons": "^5.10.2",
-    "@stoplight/elements-core": "~7.1.4",
-    "@stoplight/markdown-viewer": "^5.0.0",
+    "@stoplight/elements-core": "~7.1.5",
+    "@stoplight/markdown-viewer": "^5.1.0",
```

Comment:
Does dev-portal depend directly on markdown-viewer? 🤔

Reply:
`CustomComponentMapping` from `markdown-viewer` is used in `NodeContent`